.github/workflows/ci-workflow.yml

```
@@ -34,3 +35,22 @@ jobs:
       - name: "Website Build"
         run: |
           make ci-web
+          
+  smoke-test:
```

The way this is configured, I think it will run in parallel with `build-app` and `build-web`.  Should we run it immediately after `make ci-app` since the job name is "Build and Test API".  I think that will reduce code redundancy (i.e., setting up JDK and Python multiple times) and also ensure that we're running smoke against the actual build.

---

Initially I chose that serial route, however I opted for this configuration to parallelize the build process. I chose this direction because want the jobs to fail fast if one of the jobs failed.

---

Makes sense from a performance perspective.  But, typically the feedback loop is build, unit, integration.
It might be confusing to developers if the feedback loop results in integration tests failing earlier than the build/unit has even processed.